src/HomeTask5/Task1/MainLinkedHashSet.java

```diff
@@ -0,0 +1,37 @@
+package HomeTask5.Task1;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class MainLinkedHashSet {
+    public static void main(String[] args) {
+        Car car1 = new Car("mersedes", 80000);
+        Car car2 = new Car("audi", 60000);
+        Car car3 = new Car("bmw", 70000);
+        Car car4 = new Car("kia", 40000);
+
+        Set<Car> carLinkedHashSetList = new LinkedHashSet<Car>();
```

Comment:
can we sort LinkedHashSet?

Reply:
ArrayList<Car> array = new ArrayList<>(carLinkedHashSetList);
  Collections.sort(array);